analyzer/CMakeLists.txt

```diff
@@ -121,9 +121,14 @@ include_directories(${FRONTEND_LLVM_INCLUDE_DIR})
 
 find_package(LLVM REQUIRED)
 include_directories(SYSTEM ${LLVM_INCLUDE_DIR})
+if (LLVM_FOUND)
```

Comment:
I don't think it is worth supporting previous versions.
Could you please remove this and all the code checking for `HAS_LLVM_9`?